index.html

```diff
@@ -29,352 +32,402 @@
-        <div class="s2-container">
-          <div class="image-container2"></div>
-
-          <h1 class="newh2">Multi-Post-Stories</h1>
-          <div class="list-container">
-            <ul>
-              <li><a href="" class="text">CANOPY</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Back End Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          </div>
-          <div class="base-links flex-card">
-            <ul>
-              <li><a href="" class="base-link-text1">html</a></li>
-              <li><a href="" class="base-link-text2">css</a></li>
-              <li><a href="" class="base-link-text3">javascript</a></li>
-            </ul>
-          </div>
-          <div class="base-button flex-card">
-            <a href="">See Project</a>
           </div>
         </div>
       </div>
-      <div id="section-3">
-        <!---Begin Markup for Desktop card 3-->
-        <div class = "s3-container-dt">
-          <div class = "img-container3-dt">
-   <img src = "images/desktop-section-3.png" alt = "The book"/>  
-              </div>
-              <div class = "card-text">
-          <h2>FaceBook 360</h2>
-          <div class="list-container3-dt">
-            <ul>
-              <li><a href="" class="text">FACEBOOK</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Full Stack Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div>
-            <p>
-              Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
-            </p>
+      <section id="works">
+        <div id="section-1">
+          <!----Markup for desktop card-1-->
+          <div class="s1-container-dt">
+            <div class="img-container">
+              <img
+                src="images/desktop-section-1.png"
+                alt="Image for first card on Desktop Version: Tonic header"
+              />
+            </div>
+            <div class="card-text">
+              <h2>Tonic</h2>
+              <div class="list-container">
+                <ul>
+                  <li><a href="" class="text">CANOPY</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Back End Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-3">2015</a></li>
```

Comment:
- [ ] good job adding some of the changes required by the previous reviewer. But as she mentioned, make sure you all the alternative text `alt="..."` attributes with text describing the image. This will make your portfolio be more accessible.